bin/test.bash

```diff
@@ -23,17 +25,63 @@ test_apps_wc() {
         "${pipeline_path}/test/services/test-wc.sh" "${config[config_file_wc]}"
 }
 
-#
-# ENTRYPOINT
-#
-
-config_load "$1"
-if [[ $1 == "wc" ]]; then
-    test_apps_wc
-elif [[ $1 == "sc" ]]; then
-    test_apps_sc
-else
-    echo "ERROR:  [$1] is an invalid argument:"
-    echo "usage:   ck8s test <wc|sc>. "
-    exit 1
-fi
+function sc_help() {
+    printf "%s\n" "[Usage]: test sc [target] [ARGUMENTS]"
+    printf "%s\n" "List of targets:"
+    printf "\t%-23s %s\n" "opensearch" "Open search checks"
+    printf "\t%-23s %s\n" "fluentd" "Fluentd checks"
+    printf "\t%-23s %s\n" "opensearch-dashboards" "Opensearch dashboards checks"
+    printf "%s\n" "[NOTE] If no target is specified, the tests will go through all of them."
+    exit 0
+}
+
+function sc() {
+    if [[ ${#} == 0 ]]; then
+        test_apps_sc
+    else
+        case ${1} in
+        opensearch)
+            sc_opensearch_checks "${@:2}"
+            ;;
+        --help)
+            sc_help
+            ;;
+        *)
+            echo "unknown command: $1"
+            sc_help 1
+            exit 1
+            ;;
+        esac
+    fi
+    exit 0
+}
+
+function main_help() {
+    echo list of commands:
+    printf "%-23s %s\n" "help" "show help menu and commands"
+    printf "%-23s %s\n" "sc" "Run sc checks"
+    printf "%-23s %s\n" "wc" "Run wc checks"
+
+    exit "${1:-0}"
+}
+
+function main() {
+    if [[ ${#} == 0 ]]; then
+        main_help 0
+    fi
+
+    case ${1} in
+    sc)
+        config_load "$1"
+        with_kubeconfig "${config[kube_config_sc]}" \
+            "$1" "${@:2}"
+        ;;
+    wc)
+        config_load "$1"
+        with_kubeconfig "${config[kube_config_wc]}" \
+            "$1" "${@:2}"
```

Comment:
Don't we need to add the function `wc` for this to work? It doesn't seem to work for me at least

Reply:
Yes, I've added it in the other PR ( certificates & ingresses ), because in this one all checks are sc related

Reply:
Yes, that's the plan. 
The other PR has the current branch as target

Reply:
Right, good :+1: